doc/auxiliary-programs/src/weather-converter-program/weather-data-availability.tex

```diff
@@ -6,10 +6,18 @@ \section{Weather Data Availability}\label{weather-data-availability}
 
 % table 1
 \begin{longtable}[c]{p{1.2in}p{1.2in}p{1.2in}p{1.2in}p{1.2in}}
-\caption{Comparison of E/E with ESP-r/DOE-2/BLAST Weather Data Formats \protect \label{table:comparison-of-ee-with-esp-rdoe-2blast-weather}}\\
+
```

Comment:
@michael-okeefe @nealkruis @matthew-larson 

This is the type of change I had to make in order to get rid of the multiply defined label warnings for the tables that end up spanning two pages.  

For longtable, if we put the caption in the regular header, it will be repeated for each page the table has to span.  Because the \label is also in the caption, it will be re-declared each time as well.  If the table only hits one page, no problem, but if it spans a second page, it will be declared twice.  Or thrice.  

The solution is to put a "first header" where the caption, label, and regular header row is declared.  Then declare a regular (subsequent) header _without the label_ that is used on all subsequent pages.

It's a bit annoying, and there may be a better shorthand way to do it.  But the longtable docs are not good compared to most LaTeX package docs.  You have any thoughts moving forward?  I can try to come up with a scripted clean-up pass if needed.  But this can also wait until after release.